site/app/controllers/admin/ConfigurationController.php

```diff
@@ -150,6 +153,9 @@ public function updateConfiguration(): MultiResponse {
         ) {
             $entry = $entry === "true" ? true : false;
         }
+        elseif ($name === 'polls_enabled') {
+            $entry = $entry === "true" ? true : false;
+        }
```

Comment:
you should switch this so that `'polls_enabled'` is in the $names array above as it should do the exact same thing.